models/evc.py

```diff
@@ -399,6 +419,36 @@ def archive(self):
         """Archive this EVC on deletion."""
         self.archived = True
 
+    def _use_uni_vlan(self, uni: UNI):
+        """Use tags from UNI"""
+        if uni.user_tag is None:
+            return
+        tag = uni.user_tag.value
+        tag_type = uni.user_tag.tag_type
+        if isinstance(tag, int):
+            if not uni.interface.use_tags([tag, tag], tag_type):
+                intf = uni.interface.id
+                raise ValueError(f"Tag {tag} is not available in {intf}")
+            uni.interface.notify_interface_tags(self._controller)
```

Comment:
Same comment regarding the event notification (below).

Reply:
Deleted this one too.